cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
progrium is slated for deprecation.  I just want to make the simplest
change I can make right now.

On Fri, May 15, 2015 at 10:19 AM, Vish Kannan <notifications@github.com>
wrote:

> In cluster/addons/dns/kube2sky/Dockerfile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322#discussion_r30427329>
> :
>
> > @@ -1,4 +1,4 @@
> > -FROM scratch
>
> How about using progrium/busybox <https://github.com/progrium/busybox>
> instead? We use that for cadvisor and heapster and it is very helpful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8322/files#r30427329>
> .
>